src/components/LineChartSmallMult/LineChartSmallMult.jsx

```diff
@@ -41,6 +43,19 @@ export default class LineChartSmallMult extends PureComponent {
     metrics: [],
   }
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hover: false,
+      mouse: { x: 0, y: 0 },
+    };
```

Comment:
right. that was my thought. Going with one of the redux folks who stated
that redux isn't really for transient info.

On Tue, Sep 13, 2016 at 8:27 AM, Peter Beshai <notifications@github.com>
wrote:

> In src/components/LineChartSmallMult/LineChartSmallMult.jsx
> <https://github.com/bocoup/mlab-vis-client/pull/62#discussion_r78583296>:
>
> > @@ -41,6 +43,19 @@ export default class LineChartSmallMult extends PureComponent {
> >      metrics: [],
> >    }
> >
> > +  constructor(props) {
> > +    super(props);
> > +
> > +    this.state = {
> > +      hover: false,
> > +      mouse: { x: 0, y: 0 },
> > +    };
>
> using state! what a life. but I think it's reasonable for performance
> purposes and I don't foresee and external highlighting impacting the
> chart/the highlighting in the small mults being used externally.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bocoup/mlab-vis-client/pull/62/files/21f0478a3ee3679a3585029655ceea6a7a537026#r78583296>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAkmT3bbMcGsTbHjkr4ceCCU-v8cJyeks5qpsDlgaJpZM4J7Eoa>
> .
>